app/views/shared/_head.html.slim

```diff
@@ -6,5 +6,5 @@ head
   = csrf_meta_tags
   = stylesheet_link_tag 'application', media: 'all'
   = javascript_include_tag 'application'
-  = javascript_include_tag "http://maps.googleapis.com/maps/api/js?sensor=false&language=uk&key=#{ENV['GOOGLE_MAPS_API_KEY']}"
+  = javascript_include_tag "//maps.googleapis.com/maps/api/js?sensor=false&language=uk&key=#{ENV['GOOGLE_MAPS_API_KEY']}"
```

Comment:
add in readme doc note about ENV keys
also probably better to use secrets (but for heroku ENV is ok ... so it's optional)